Lesson04.Methods/Lesson04.Methods/Program.cs

```diff
@@ -116,5 +119,64 @@ static int SumNumbers(int a, int b)
 
             return sum;
         }
+
+        //------------------------------HOMEWORK---------------------------------//
+
+        static void Master()
+        {
+            Console.Write("Input first number: ");
+            int a = int.Parse(Console.ReadLine());
+            Console.Write("Input second number: ");
+            int b = int.Parse(Console.ReadLine());
+
+            Console.Write("Input string: ");
+            string x = Console.ReadLine();
+
+            MaxMinValue(54, 76);
+            MaxMinValue(3, 43, 23);
+            MaxMinValue(4.3, 9.23, 1.11, 13.32);
+
+            Repeat(x);
+
+            if (TrySumIfOdd(a, b, out int sum))
+            {
+                Console.WriteLine("\nParity \nSum is: " + sum);
+            }
+
+            else
+            {
+                Console.WriteLine("\nNot Parity \nSum is: " + sum);
+            }
+
+        }
+        static void MaxMinValue(int x, int y)
+        {
+            Console.WriteLine("Max value is: " + Math.Max(x, y));
```

Comment:
try to follow single responsibility principle. Single method should has one responsibility

Reply:
Can you a little bit explain about how correct implement SRP in this part of code?